### ISSUE TITLE
Debian 8.8.0 "Jessie" guest on WIN10: Makefile:574 ../lib/stubs/.deps/livmtools_la-stub-log.Plo: No such file or directory

### ISSUE THREAD

**Issue by reforge-reprap** [vmware/open-vm-tools#171]
Hi, right to the point.
 I've installed all the dependecies. That's my 'autoreconf -i':
```
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:1005: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:4100: _LT_LINKER_SHLIBS is expanded from...
m4/libtool.m4:5167: _LT_LANG_C_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:4100: _LT_LINKER_SHLIBS is expanded from...
m4/libtool.m4:5167: _LT_LANG_C_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:1005: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:5274: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:787: _LT_LANG is expanded from...
m4/libtool.m4:770: LT_LANG is expanded from...
m4/libtool.m4:798: _LT_LANG_DEFAULT_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:1005: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:4100: _LT_LINKER_SHLIBS is expanded from...
m4/libtool.m4:5167: _LT_LANG_C_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:4100: _LT_LINKER_SHLIBS is expanded from...
m4/libtool.m4:5167: _LT_LANG_C_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:1005: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:5274: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:787: _LT_LANG is expanded from...
m4/libtool.m4:770: LT_LANG is expanded from...
m4/libtool.m4:798: _LT_LANG_DEFAULT_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:1005: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:4100: _LT_LINKER_SHLIBS is expanded from...
m4/libtool.m4:5167: _LT_LANG_C_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:4100: _LT_LINKER_SHLIBS is expanded from...
m4/libtool.m4:5167: _LT_LANG_C_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
configure.ac:280: warning: AC_LANG_CONFTEST: no AC_LANG_SOURCE call detected in body
../../lib/autoconf/lang.m4:193: AC_LANG_CONFTEST is expanded from...
../../lib/autoconf/general.m4:2661: _AC_LINK_IFELSE is expanded from...
../../lib/autoconf/general.m4:2678: AC_LINK_IFELSE is expanded from...
m4/libtool.m4:1005: _LT_SYS_MODULE_PATH_AIX is expanded from...
m4/libtool.m4:5274: _LT_LANG_CXX_CONFIG is expanded from...
m4/libtool.m4:787: _LT_LANG is expanded from...
m4/libtool.m4:770: LT_LANG is expanded from...
m4/libtool.m4:798: _LT_LANG_DEFAULT_CONFIG is expanded from...
m4/libtool.m4:138: _LT_SETUP is expanded from...
m4/libtool.m4:67: LT_INIT is expanded from...
m4/libtool.m4:102: AC_PROG_LIBTOOL is expanded from...
configure.ac:280: the top level
libtoolize: putting auxiliary files in AC_CONFIG_AUX_DIR, `config'.
libtoolize: copying file `config/ltmain.sh'
libtoolize: putting macros in AC_CONFIG_MACRO_DIR, `m4'.
libtoolize: copying file `m4/libtool.m4'
libtoolize: copying file `m4/ltoptions.m4'
libtoolize: copying file `m4/ltsugar.m4'
libtoolize: copying file `m4/ltversion.m4'
libtoolize: copying file `m4/lt~obsolete.m4'
configure.ac:261: installing 'config/compile'
configure.ac:74: installing 'config/config.guess'
configure.ac:74: installing 'config/config.sub'
configure.ac:238: installing 'config/install-sh'
configure.ac:238: installing 'config/missing'
checkvm/Makefile.am: installing 'config/depcomp'
lib/guestRpc/Makefile.am:26: warning: 'CFLAGS' is a user variable, you should not override it;
lib/guestRpc/Makefile.am:26: use 'AM_CFLAGS' instead
libguestlib/Makefile.am:54: warning: 'CFLAGS' is a user variable, you should not override it;
libguestlib/Makefile.am:54: use 'AM_CFLAGS' instead
tests/testDebug/Makefile.am:44: warning: 'CFLAGS' is a user variable, you should not override it;
tests/testDebug/Makefile.am:44: use 'AM_CFLAGS' instead
tests/testPlugin/Makefile.am:44: warning: 'CFLAGS' is a user variable, you should not override it;
tests/testPlugin/Makefile.am:44: use 'AM_CFLAGS' instead
```
Here the './configure' output:
```
checking build system type... x86_64-unknown-linux-gnu
checking host system type... x86_64-unknown-linux-gnu
checking for a BSD-compatible install... /usr/bin/install -c
checking whether build environment is sane... yes
checking for a thread-safe mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for style of include used by make... GNU
checking for gcc... gcc
checking whether the C compiler works... yes
checking for C compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether we are using the GNU C compiler... yes
checking whether gcc accepts -g... yes
checking for gcc option to accept ISO C89... none needed
checking whether gcc understands -c and -o together... yes
checking dependency style of gcc... gcc3
checking how to run the C preprocessor... gcc -E
checking for gcc... (cached) gcc
checking whether we are using the GNU C compiler... (cached) yes
checking whether gcc accepts -g... (cached) yes
checking for gcc option to accept ISO C89... (cached) none needed
checking whether gcc understands -c and -o together... (cached) yes
checking dependency style of gcc... (cached) gcc3
checking for g++... g++
checking whether we are using the GNU C++ compiler... yes
checking whether g++ accepts -g... yes
checking dependency style of g++... gcc3
checking for a sed that does not truncate output... /bin/sed
checking whether ln -s works... yes
checking how to print strings... printf
checking for a sed that does not truncate output... (cached) /bin/sed
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
checking for fgrep... /bin/grep -F
checking for ld used by gcc... /usr/bin/ld
checking if the linker (/usr/bin/ld) is GNU ld... yes
checking for BSD- or MS-compatible name lister (nm)... /usr/bin/nm -B
checking the name lister (/usr/bin/nm -B) interface... BSD nm
checking the maximum length of command line arguments... 1572864
checking whether the shell understands some XSI constructs... yes
checking whether the shell understands "+="... yes
checking how to convert x86_64-unknown-linux-gnu file names to x86_64-unknown-linux-gnu format... func_convert_file_noop
checking how to convert x86_64-unknown-linux-gnu file names to toolchain format... func_convert_file_noop
checking for /usr/bin/ld option to reload object files... -r
checking for objdump... objdump
checking how to recognize dependent libraries... pass_all
checking for dlltool... no
checking how to associate runtime and link libraries... printf %s\n
checking for ar... ar
checking for archiver @FILE support... @
checking for strip... strip
checking for ranlib... ranlib
checking command to parse /usr/bin/nm -B output from gcc object... ok
checking for sysroot... no
checking for mt... mt
checking if mt is a manifest tool... no
checking for ANSI C header files... yes
checking for sys/types.h... yes
checking for sys/stat.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for memory.h... yes
checking for strings.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for unistd.h... yes
checking for dlfcn.h... yes
checking for objdir... .libs
checking if gcc supports -fno-rtti -fno-exceptions... no
checking for gcc option to produce PIC... -fPIC -DPIC
checking if gcc PIC flag -fPIC -DPIC works... yes
checking if gcc static flag -static works... yes
checking if gcc supports -c -o file.o... yes
checking if gcc supports -c -o file.o... (cached) yes
checking whether the gcc linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking whether -lc should be explicitly linked in... no
checking dynamic linker characteristics... GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking whether stripping libraries is possible... yes
checking if libtool supports shared libraries... yes
checking whether to build shared libraries... yes
checking whether to build static libraries... yes
checking how to run the C++ preprocessor... g++ -E
checking for ld used by g++... /usr/bin/ld -m elf_x86_64
checking if the linker (/usr/bin/ld -m elf_x86_64) is GNU ld... yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking for g++ option to produce PIC... -fPIC -DPIC
checking if g++ PIC flag -fPIC -DPIC works... yes
checking if g++ static flag -static works... yes
checking if g++ supports -c -o file.o... yes
checking if g++ supports -c -o file.o... (cached) yes
checking whether the g++ linker (/usr/bin/ld -m elf_x86_64) supports shared libraries... yes
checking dynamic linker characteristics... (cached) GNU/Linux ld.so
checking how to hardcode library paths into programs... immediate
checking for pkg-config... yes
checking for X... libraries , headers 
checking for gethostbyname... yes
checking for connect... yes
checking for remove... yes
checking for shmat... yes
checking for IceConnectionNumber in -lICE... yes
checking libintl.h usability... yes
checking libintl.h presence... yes
checking for libintl.h... yes
checking for libmspack >= 0.0.20040308alpha (via pkg-config)... yes
checking for glib-2.0 >= 2.14.0 (via pkg-config)... yes
checking for gmodule-2.0 >= 2.14.0 (via pkg-config)... yes
checking for gobject-2.0 >= 2.14.0 (via pkg-config)... yes
checking for gthread-2.0 >= 2.14.0 (via pkg-config)... yes
checking for glib-genmarshal... yes
checking for fuse (via pkg-config)... no
configure: WARNING: Fuse is missing, vmblock-fuse/vmhgfs-fuse will be disabled.
checking security/pam_appl.h usability... yes
checking security/pam_appl.h presence... yes
checking for security/pam_appl.h... yes
checking for pam_start in -lpam... yes
checking for BIO_new_file in -lssl... no
checking for openssl >= 1.0.1 (via pkg-config)... yes
checking for strlen in -lxerces-c... yes
checking for strlen in -lxml-security-c... yes
checking CUnit/CUnit.h usability... yes
checking CUnit/CUnit.h presence... yes
checking for CUnit/CUnit.h... yes
checking for CU_initialize_registry in -lcunit... yes
checking for XeviQueryVersion in -lXext... yes
checking for X11/extensions/extutil.h... yes
checking for XineramaQueryVersion in -lXinerama... yes
checking for XOpenDevice in -lXi... yes
checking for XRenderQueryVersion in -lXrender... yes
checking for XRRQueryVersion in -lXrandr... yes
checking for XTestQueryExtension in -lXtst... yes
checking for SmcOpenConnection in -lSM... yes
checking X11/SM/SMlib.h usability... yes
checking X11/SM/SMlib.h presence... yes
checking for X11/SM/SMlib.h... yes
checking X11/ICE/ICElib.h usability... yes
checking X11/ICE/ICElib.h presence... yes
checking for X11/ICE/ICElib.h... yes
checking for XCompositeQueryExtension in -lXcomposite... yes
checking X11/extensions/Xcomposite.h usability... yes
checking X11/extensions/Xcomposite.h presence... yes
checking for X11/extensions/Xcomposite.h... yes
checking for gtk+-2.0 >= 2.4.0 (via pkg-config)... yes
checking gtkmm.h usability... no
checking gtkmm.h presence... no
checking for gtkmm.h... no
checking for gtkmm-2.4 >= 2.4.0 (via pkg-config)... yes
checking for sigc++-2.0 >= 2.5.1 (via pkg-config)... no
checking for crypt in -lcrypt... yes
checking for dlopen... no
checking for dlopen in -ldl... yes
checking for ecvt... yes
checking for fcvt... yes
checking for mkdtemp... yes
checking for pthread_mutex_init in -lpthread... yes
checking for libprocps (via pkg-config)... yes
checking for dumbnet-config... /usr/bin/dumbnet-config
checking for g++... g++
checking for icu-config... /usr/bin/icu-config
checking for ucasemap_utf8ToTitle in ICU... yes
checking for rpcgen... /usr/bin/rpcgen
checking crypt.h usability... yes
checking crypt.h presence... yes
checking for crypt.h... yes
checking for inttypes.h... (cached) yes
checking for stdint.h... (cached) yes
checking for stdlib.h... (cached) yes
checking wchar.h usability... yes
checking wchar.h presence... yes
checking for wchar.h... yes
checking sys/inttypes.h usability... no
checking sys/inttypes.h presence... no
checking for sys/inttypes.h... no
checking sys/io.h usability... yes
checking sys/io.h presence... yes
checking for sys/io.h... yes
checking sys/param.h usability... yes
checking sys/param.h presence... yes
checking for sys/param.h... yes
checking sys/sysinfo.h usability... yes
checking sys/sysinfo.h presence... yes
checking for sys/sysinfo.h... yes
checking for sys/types.h... (cached) yes
checking for sys/user.h... yes
checking sys/vfs.h usability... yes
checking sys/vfs.h presence... yes
checking for sys/vfs.h... yes
checking syslimits.h usability... no
checking syslimits.h presence... no
checking for syslimits.h... no
checking unwind.h usability... yes
checking unwind.h presence... yes
checking for unwind.h... yes
checking for wchar.h... (cached) yes
checking linux/unistd.h usability... yes
checking linux/unistd.h presence... yes
checking for linux/unistd.h... yes
checking for X11/extensions/panoramiXproto.h... yes
checking for ecvt in -lc... yes
checking for stdbool.h that conforms to C99... yes
checking for _Bool... yes
checking for an ANSI C-conforming const... yes
checking for uid_t in sys/types.h... yes
checking for inline... inline
checking for mode_t... yes
checking for off_t... yes
checking for pid_t... yes
checking for size_t... yes
checking for struct stat.st_rdev... yes
checking whether time.h and sys/time.h may both be included... yes
checking whether struct tm is in sys/time.h or time.h... time.h
checking for working volatile... yes
checking for GCC flag -Wno-pointer-sign... yes
checking for GCC flag -Wno-unused-value... yes
checking for GCC flag -fno-strict-aliasing... yes
checking for GCC flag -Wno-unknown-pragmas... yes
checking for GCC flag -Wno-uninitialized... yes
checking for GCC flag -Wno-deprecated-declarations... yes
checking for GCC flag -Wno-unused-const-variable... yes
checking for GCC flag -fvisibility... yes
checking for GCC flag -Wno-unused-but-set-variable... yes
checking for doxygen... yes
checking for dot... no
checking for mscgen... no
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating lib/Makefile
config.status: creating lib/appUtil/Makefile
config.status: creating lib/auth/Makefile
config.status: creating lib/backdoor/Makefile
config.status: creating lib/asyncsocket/Makefile
config.status: creating lib/sslDirect/Makefile
config.status: creating lib/pollGtk/Makefile
config.status: creating lib/poll/Makefile
config.status: creating lib/dataMap/Makefile
config.status: creating lib/hashMap/Makefile
config.status: creating lib/dict/Makefile
config.status: creating lib/dynxdr/Makefile
config.status: creating lib/err/Makefile
config.status: creating lib/file/Makefile
config.status: creating lib/foundryMsg/Makefile
config.status: creating lib/glibUtils/Makefile
config.status: creating lib/guestApp/Makefile
config.status: creating lib/guestRpc/Makefile
config.status: creating lib/hgfs/Makefile
config.status: creating lib/hgfsBd/Makefile
config.status: creating lib/hgfsHelper/Makefile
config.status: creating lib/hgfsServer/Makefile
config.status: creating lib/hgfsServerManagerGuest/Makefile
config.status: creating lib/hgfsServerPolicyGuest/Makefile
config.status: creating lib/hgfsUri/Makefile
config.status: creating lib/impersonate/Makefile
config.status: creating lib/lock/Makefile
config.status: creating lib/message/Makefile
config.status: creating lib/misc/Makefile
config.status: creating lib/netUtil/Makefile
config.status: creating lib/nicInfo/Makefile
config.status: creating lib/panic/Makefile
config.status: creating lib/panicDefault/Makefile
config.status: creating lib/procMgr/Makefile
config.status: creating lib/rpcChannel/Makefile
config.status: creating lib/rpcIn/Makefile
config.status: creating lib/rpcOut/Makefile
config.status: creating lib/rpcVmx/Makefile
config.status: creating lib/slashProc/Makefile
config.status: creating lib/string/Makefile
config.status: creating lib/stubs/Makefile
config.status: creating lib/syncDriver/Makefile
config.status: creating lib/system/Makefile
config.status: creating lib/unicode/Makefile
config.status: creating lib/user/Makefile
config.status: creating lib/vmCheck/Makefile
config.status: creating lib/vmSignal/Makefile
config.status: creating lib/wiper/Makefile
config.status: creating lib/xdg/Makefile
config.status: creating services/Makefile
config.status: creating services/vmtoolsd/Makefile
config.status: creating services/plugins/Makefile
config.status: creating services/plugins/desktopEvents/Makefile
config.status: creating services/plugins/dndcp/Makefile
config.status: creating services/plugins/grabbitmqProxy/Makefile
config.status: creating services/plugins/guestInfo/Makefile
config.status: creating services/plugins/hgfsServer/Makefile
config.status: creating services/plugins/powerOps/Makefile
config.status: creating services/plugins/resolutionSet/Makefile
config.status: creating services/plugins/timeSync/Makefile
config.status: creating services/plugins/vix/Makefile
config.status: creating services/plugins/vmbackup/Makefile
config.status: creating services/plugins/deployPkg/Makefile
config.status: creating vmware-user-suid-wrapper/Makefile
config.status: creating toolbox/Makefile
config.status: creating hgfsclient/Makefile
config.status: creating hgfsmounter/Makefile
config.status: creating checkvm/Makefile
config.status: creating rpctool/Makefile
config.status: creating guestproxycerttool/Makefile
config.status: creating vgauth/Makefile
config.status: creating vgauth/lib/Makefile
config.status: creating namespacetool/Makefile
config.status: creating vgauth/cli/Makefile
config.status: creating vgauth/service/Makefile
config.status: creating libguestlib/Makefile
config.status: creating libguestlib/vmguestlib.pc
config.status: creating libDeployPkg/Makefile
config.status: creating libDeployPkg/libDeployPkg.pc
config.status: creating libhgfs/Makefile
config.status: creating libvmtools/Makefile
config.status: creating common-agent/Cpp/Framework/Makefile
config.status: creating common-agent/Cpp/Communication/Makefile
config.status: creating common-agent/Cpp/InternalProviders/Makefile
config.status: creating common-agent/Cpp/ManagementAgent/Makefile
config.status: creating common-agent/Cpp/ProviderFx/Makefile
config.status: creating common-agent/input/Makefile
config.status: creating common-agent/input/invokers/Makefile
config.status: creating common-agent/input/providerReg/Makefile
config.status: creating common-agent/etc/Makefile
config.status: creating common-agent/etc/config/Makefile
config.status: creating common-agent/etc/scripts/Makefile
config.status: creating common-agent/etc/install/Makefile
config.status: creating xferlogs/Makefile
config.status: creating modules/Makefile
config.status: creating vmblock-fuse/Makefile
config.status: creating vmhgfs-fuse/Makefile
config.status: creating vmblockmounter/Makefile
config.status: creating tests/Makefile
config.status: creating tests/vmrpcdbg/Makefile
config.status: creating tests/testDebug/Makefile
config.status: creating tests/testPlugin/Makefile
config.status: creating tests/testVmblock/Makefile
config.status: creating docs/Makefile
config.status: creating docs/api/Makefile
config.status: creating scripts/Makefile
config.status: creating scripts/build/rpcgen_wrapper.sh
config.status: creating udev/Makefile
config.status: executing depfiles commands
config.status: executing libtool commands
```

and the last few lines of the 'make check':
```
make[2]: Leaving directory '/open-vm-tools-master/open-vm-tools/lib/xdg'
make[2]: Entering directory '/open-vm-tools-master/open-vm-tools/lib'
make[2]: Nothing to be done for 'check-am'.
make[2]: Leaving directory '/open-vm-tools-master/open-vm-tools/lib'
make[1]: Leaving directory '/open-vm-tools-master/open-vm-tools/lib'
Making check in libvmtools
make[1]: Entering directory '/open-vm-tools-master/open-vm-tools/libvmtools'
Makefile:574: ../lib/stubs/.deps/libvmtools_la-stub-log.Plo: No such file or directory
make[1]: *** No rule to make target '../lib/stubs/.deps/libvmtools_la-stub-log.Plo'.  Stop.
make[1]: Leaving directory '/open-vm-tools-master/open-vm-tools/libvmtools'
Makefile:523: recipe for target 'check-recursive' failed
make: *** [check-recursive] Error 1
```
I'm currently using open-vm-tools 9.4.6.33107 (build 1770165) as reported by 'vmware-toolbox-cmd -v'

Help appreciated. Thanks.

**Comment by oliverkurth** [comment]
This is weird, I haven't seen this before. Which branch did you check out?

**Comment by johnwvmw** [comment]
Thanks Oliver for taking a look at this.


Get well soon

John

________________________________

**Comment by reforge-reprap** [comment]
Thanks for the help. If you meant which branch of open-vm-tools I'm using then it's your last commit as I downloaded the sources yesterday - ver. 10.1.5.
I omitted some details, here some more:
the box I've virtualized is isolated - it's not joining any network. The packages I've installed come from the first three official Debian DVDs and from the Debian repository when some packages needed weren't released on the discs, all of which I've downloaded from an entirely different machine. Therefore the dependencies installation has been done entirely off the wire. 
Here's what I've pulled, please note that the list order respects the execution time when the errors were thrown; naturally all the deb files where installed manually:
  ```
   autoconf 2.69-8 (installs automake 1:1.14.1-4+deb8ul as a dep) 
   init-system-helpers 
   libdumbnet1 
   ethtool
   zerofree
   libtool (or 'autoreconf -i' command will fail to find libtoolize)
   open-vm-tools-dkms (or ./configure won't find the kernel headers in /lib/modules/3.16.0-4-amd64/include )
   pkg-config
   libmspack0:amd64
   libmspack-dev_0.5-1_amd64.deb (or configure: error: mspack >= 0.0.20040308alpha is required)
   glib2.0
   libpam0g-dev
   openssl_1.0.1t-1+deb8u6_amd64.deb (the installed version 1.0.0 was conflicting)
   libssl1.0.2_1.0.2l-2_amd64.deb
   libssl-dev
   libxerces-c-dev
   libxml-security-c17_1.7.2-3+b1_amd64.deb
   libxml-security-c-dev_1.7.2-3+b1_amd64.deb
   libx11-dev
   libXext-dev
   libxinerama1_1.1.3-1+b1_amd64.deb
   x11proto-xinerama-dev_1.2.1-2_all.deb
   libxinerama-dev_1.1.3-1+b1_amd64.deb
   x11proto-core-dev_7.0.26-1_all.deb
   x11proto-xext-dev_7.3.0-1_all.deb
   x11proto-fixes-dev_5.0-2_all.deb
   libxfixes-dev_5.0.1-2+b2_amd64.deb
   libxi-dev_1.7.4-1+b2_amd64.deb
   x11proto-render-dev_0.11.1-2_all.deb
   libxrender1_0.9.8-1+b1_amd64.deb
   libxrender-dev_0.9.8-1+b1_amd64.deb
   x11proto-randr-dev_1.4.0-2_all.deb
   libxrandr2_1.4.2-1+b1_amd64.deb
   libxrandr-dev_1.4.2-1+b1_amd64.deb
   libxtst6_1.2.2-1+b1_amd64.deb
   x11proto-record-dev_1.14.2-1_all.deb
   libxtst-dev_1.2.2-1+b1_amd64.deb
   gtk2.0
   gtkmm-2.4
   libprocps3_3.3.9-9_amd64.deb
   libprocps3-dev_3.3.9-9_amd64.deb
   libdnet
   libdnet-dev_2.64_amd64.deb
   libdumbnet-dev_1.12-6_amd64.deb
   libgtkmm-2.4-1c2a_2.24.4-1.1_amd64.deb
   libgtkmm-2.4-dev_2.24.4-1.1_amd64.deb
   
   doxygen (or configure: WARNING: doxygen not found; API documentation will not be generated.)
   libcunit1_2.1-2.dfsg-2_amd64.deb (or configure: WARNING: CUnit not found, tests won't be compiled.)
   libcunit1-doc_2.1-2.dfsg-2_all.deb
   libcunit1-dev_2.1-2.dfsg-2_amd64.deb
```
Most packages I manually installed are for Debian stable, but few I had to pull from 'Sid', e.g. openssl.

**Comment by oliverkurth** [comment]
If you pulled packages from various sources, mixing Debian/stable and sid, this very much looks like a broken system to me. I would try to use a clean system, preferably Debian/stable since this is better tested. Debian/sid is a moving target, but it should work most of the time.

Also, Debian has an open-vm-tools package. Is there a reason you are not using that?

**Comment by reforge-reprap** [comment]
Let's pass over to what may or may not constitute a "broken" system in general terms, and pass over to the fact that you are suggesting me to use a very old version of open-vm-tools "that Debian has" - that's the 9.4.6 which I ALREADY said I am using - just because the latest doesn't seem to work, and pass over to what you think may constitute a "clean system", I did take seriously the bit about "pulling packages from various sources".

First off, the sources are only two: the Jessie repository and the Sid one. Secondly I've run a check on *all* the packages I've manually installed - which are the only ones could have been pulled from Sid - and they *all* come from the Jessie repository as well, but one:
`libssl1.0.2 (1.0.2l-2) `
which is shared in Stretch too. I had to pulled this one from Sid, because it was a requirement. The stable version of it - stable enough to be included in Debian Jessie, that is - IT IS NOT WORKING with the master branch, 10.1.5. I also reported that already.
All in all, it seems it isn't my system that is "broken".

Let's summarize what I've done.
I've installed an all Debian "stable" distro, but one package and I cannot manage to compile open-vm-tools even so. 
Just to be 100% clear:
Jessie is the codename for Debian 8 Stable, "Stretch" for the Testing version and "Sid" is the "Unstable". Stretch is going to become the next Debian distro so a reasonable amount of packages already work together. While Sid, as you put it, it is more complicated to manage, as all the rolling releases are, but it should still run for the most part. Let's at least agree on these. lol

**Comment by oliverkurth** [comment]
We do regular tests to  build open-vm-tools in Debian, and strive to cover as many distributions as possible. But we cannot cover all possible permutations of mixes between stable and unstable distributions.

Just to make sure we didn't miss anything, I manually retested the master branch with current Debian Jessie, Stretch and Sid. All tests completed successfully. Since you mentioned issues with libssl1.0.2 in Jessie, I upgraded that to the latest in Jessie (1.0.1t-1+deb8u6). I also removed the package libfuse-dev, since you do not seem to have this installed, and it still builds. There does not seem to be any requirement to add packages from sid to jessie.

To build open-vm-tools in Debian, you can install dependencies with these commands - this should work in Jessie, Stretch and Sid:
```
  apt-get build-dep -y open-vm-tools && \
  apt-get install -y libmspack-dev libssl-dev libxerces-c-dev libxml-security-c-dev libprocps3-dev
```

If you just want latest open-vm-tools in Jessie, you can also install using jessie-backports, see https://packages.debian.org/search?keywords=open-vm-tools . This looks like the easiest path.

If you still see issues building, please attach log files with the output of autoreconf, configure and make, and attach those files with you comment (no need to copy inline). You can use tee and include the error output, for example:

`./configure 2>&1 | tee configure.out`

**Comment by reforge-reprap** [comment]
Thank you for taking the time to answer me again.
- You mentioned the devel package of libssl - libssl-dev_1.0.1t-1+deb8u6 is installed on my box already. The share libraries of that Jessie package libssl1.0.0 is too old for ovt1015, so it was required of me the 1.0.2. Stretch had it. I reported it for completeness' sake as I never thought it was the reason why I wasn't able to compile ovt.

- I don't understand which part of "...the box I've virtualized is isolated - it's not joining any network,[...]" is confusing. I'm working on a non-networked machine -as in 'no internet', hence I can't populate sources.list with link calls to Debian repositories, can't I?. 
I could use apt-cdrom, but this route would require of me downloading more official DVDs using Jigdo as the first three don't contain the sources I need, some of which you pointed out. 
I'm left with the lovely option of using apt-offline on a Windows machine with internet connection. 

My aim, clearly, is to compile ovt on my own. I'm aware I don't need to do that to install the ovt package lol I downloaded an ovt binary package and with the dependencies I've already manually installed, ovt1015 has been working. Which accidentally tells me I have all the packages needed on the system by now. 
It's the compiling process that halts with "a weird" error, as you put it.

Anyway I've moved ahead on to Debian 9.0.0 just released. Hopefully I will be able to compile offline on Stretch. Keep you posted if it's of any interest or feel free to close the thread otherwise. 
Thanks for all the retesting you've done because of me.
